pkg/template/templates/bash/bash.tmpl

```diff
@@ -24,7 +24,7 @@ Documentation=https://www.minecraft.net/en-us/download/server
 WorkingDirectory=/minecraft
 Type=simple
 {{- if ne .Spec.Minecraft.Edition "bedrock" }}
-ExecStart=/usr/bin/java -Xmx{{.Spec.Minecraft.Java.Xmx}} -Xms{{.Spec.Minecraft.Java.Xms}} -jar server.jar nogui
+ExecStart=/usr/bin/java -Xmx{{.Spec.Minecraft.Java.Xmx}} -Xms{{.Spec.Minecraft.Java.Xms}}{{range .Spec.Minecraft.Java.Options }} {{.}}{{end}} -jar server.jar nogui
```

Comment:
You should add this to the cloud-config.yaml.tmpl too... (for server and proxy too)

Reply:
Added in new commit, thanks for the catch!